third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -895,7 +1023,7 @@ impl Loader {
 
         self.check_natives(&module)
             .map_err(expect_no_verification_errors)?;
-        Ok(module)
+        Ok((size, module))
```

Comment:
nit: it might be nicer to have module first and then size (like this is a module and its size)